source/documentation/how-to/hosted-engine.html.md

```diff
@@ -35,7 +35,7 @@ Greg Padgett <gpadgett@redhat.com>, Martin Sivak <msivak@redhat.com>
 
 Assuming you're using ovirt RPMs, you should start with install and deploy:
 
-         # yum install ovirt-hosted-engine-setup
+         # yum install ovirt-hosted-engine-setup ovirt-engine-appliance
```

Comment:
Maybe worth to mention that installing the appliance using the rpm is optional and that the package can be removed after the deployment saving ~1gb of disk space.